src/tests/system/tests/test_ldap.py

```diff
@@ -538,3 +538,56 @@ def test_ldap__empty_attribute(client: Client, ldap: LDAP):
     for grp in ["Group_1", "Group_2"]:
         assert client.tools.getent.group(grp) is not None
     assert client.auth.ssh.password(user.name, "Secret123"), "User login failed!"
+
+
+@pytest.mark.importance("low")
+@pytest.mark.topology(KnownTopology.LDAP)
+def test_netgroup_search_base(client: Client, provider: LDAP):
+    """
+    :title: ldap search base does not fully limit the Netgroup search base
+    :setup:
+        1. Netgroups are created in different ous
+        2. Members are added to netgroups
+        3. The ldap search base is reconfigured to only include ou=Netgroup1
+    :steps:
+        1. The provider role is ldap, "Seceng" should still be resolvable,
+            validating that the ldap search base does not disrupt resolution of netgroups in ou1.
+    :expectedresults:
+        1. id look up should success
```

Comment:
Please fix the steps to match meaningfully the asserts.  setup 3. is probably test step as You are doing it in the middle of asserts.